tests/services/test_record_service_tasks.py

```diff
@@ -0,0 +1,74 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2020-2021 CERN.
+# Copyright (C) 2020-2021 Northwestern University.
+#
+# Invenio-Drafts-Resources is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see LICENSE file for more
+# details.
+
+"""Service tests.
+
+Test to add:
+- Read a tombstone page
+- Read with missing permissions
+- Read with missing pid
+"""
+
+from io import BytesIO
+
+import pytest
+from invenio_db import db
+from mock_module.service import ServiceConfig
+
+from invenio_drafts_resources.services import RecordService
+from invenio_drafts_resources.services.records.tasks import cleanup_drafts
+
+
+#
+# Fixtures
+#
+@pytest.fixture()
+def input_data(input_data):
+    """Enable files."""
+    input_data['files']['enabled'] = True
```

Comment:
If you set files to `False`, you won't need all the files mechanics and will simplify the tests...
```suggestion
    input_data['files']['enabled'] = False
```

It is also to be more concise and *only* test what is at stake, i.e. cleanup (but not files, which is tested in `test_record_service_files.py`.